nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ControlRate.java

```diff
@@ -381,6 +392,14 @@ public boolean tryAdd(final long value) {
 
     private class ThrottleFilter implements FlowFileFilter {
 
+        private final long flowFilesPerTrigger;
+        private final AtomicLong flowFilesFiltered = new AtomicLong(0L);
```

Comment:
This filter is not thread-safe... don't think we need an AtomicLong here. Can just use an int.